src/route-handlers/list-workflows/list-workflows.ts

```diff
@@ -60,7 +60,7 @@ export async function listWorkflows(
   } catch (e) {
     return NextResponse.json(
       {
-        message: 'Error fetching workflows',
+        message: e instanceof Error ? e.message : 'Error fetching workflows',
```

Comment:
This can be concerning.
From what i understand this can expose all errors, even stacktrace.

Reply:
Agreed. I've fixed this to use the message only if the error thrown is a GRPCError.